### PR TITLE
Various fix for master (after list, form, kanban ref)

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/views/form/form_controller.scss [hunk 1]
```diff
@@ -174,7 +177,7 @@ $o-form-label-margin-right: 0px;
     }
 
     // No sheet
-    .o_form_nosheet {
+    &.o_form_nosheet {
```
Comment (Polymorphe57) [inline addons/web/static/src/views/form/form_controller.scss:180]: + ,o_form_nosheet

File: addons/web/static/src/views/form/form_controller.scss [hunk 2]
```diff
@@ -60,6 +60,9 @@ $o-form-label-margin-right: 0px;
             }
         }
     }
+    .oe_inline > * {
```
[the remark this answers is not in the document]
Comment (Polymorphe57) [reply addons/web/static/src/views/form/form_controller.scss:63]: move to l55

**Comment by sdegueldre** [comment]
@robodoo r+

**Comment by sdegueldre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**